tracer/tracer_test.go

```diff
@@ -113,6 +116,50 @@ func TestTracerEnabledAgain(t *testing.T) {
 	assert.Equal(tracer.buffer.Len(), 1)
 }
 
+// Run TestTraceEnabledVar in a separate process since it messes with global state.
```

Comment:
Yeah that's the problem. Using the ``init()`` (that is usually a good approach for this kind of stuff) requires to start the process again to test the env variable. If we use a simpler approach, collecting environment variables while building the tracer (I mean the ``NewTracer()`` func), it will make this test easier and without race condition for the ``enabled`` field.

Will think about it a little more today. Thank you!